tests/Pyxsim/xmostest_subprocess.py

```diff
@@ -85,7 +85,9 @@ def platform_is_windows():
     use_shell = True
     # Windows version of Python 2.7 doesn't support SIGINT
     if sys.version_info < (3, 0):
-        raise TestError("Doesn't support Python version < 3, please upgrade to Python 3 or higher.")
+        raise TestError(
+            "Doesn't support Python version < 3, please upgrade to Python 3 or higher."
+        )
```

Comment:
Isn't this true for all platforms, now? Maybe move this to somewhere generic?

Reply:
I think this is something @kevinyeungxmos put in based on a comment from @mbanth (my changes were just auto-formatting via black)

Reply:
The whole pyxsim folder needs moving to another repo.